test/test_api.cpp

```diff
@@ -12,12 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <gtest/gtest.h>
```

Comment:
`gtest.h` is C++ and should therefore be included after C headers like `sys/resource.h`.